src/views/TabView/TabBarBottom.js

```diff
@@ -89,6 +89,18 @@ export default class TabBarBottom extends PureComponent<DefaultProps, Props, voi
         </Animated.Text>
       );
     }
+    
+    if (typeof label === 'function') {
+      const st = [styles.label, { color: inactiveTintColor }, labelStyle];
+      const params = {
+        ...scene,
+        defaultStyles: StyleSheet.flatten(st)
```

Comment:
That sounds like very specific usecase. You already receive the `tintColor` as a prop, so you can write your own label component which animates between the colors when it changes. Also there's no reason to flatten the style here.